includes/Renderer/Renderer.php

```diff
@@ -0,0 +1,265 @@
+<?php
+/**
+ * Class Renderer
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Renderer;
```

Comment:
So with this now w'd have three namespaces: `Story_Renderer`, `Stories_Renderer`, `Renderer` 😕 

That makes the existing class structure even more confusing.

Especially since this new Renderer class in this new namespace does three different things:

1. Display Embed/Image in archives/search view
2. Display full story for single view
3. Filter oEmbed logic and template

These three could easily be three separate classes, perhaps in a `Frontend` namespace.

That would help reduce confusion.

WDYT?

Reply:
Way ahead of you on this one https://github.com/google/web-stories-wp/pull/7288

This makes it. 

`Renderer`
`Renderer\Story`
`Renderer\Stories`

Reply:
Cool, renaming those two makes sense. Much better already.

But still seems like this `Renderer` class here could be split up in 3 and moved to a separate `Frontend` namespace.

So:

`Frontend` (with `oEmbed`, `Archives`, `Single` or something)
`Renderer\Story`
`Renderer\Stories`

Reply:
I agree with breaking up the renderer class, it is a big class that does a lot of things. 

Not sure about `Frontend` namespace. A render class could easily be used to render something that is not on the FE. I would say that embed / feeds are not exactly front end..

Reply:
OK, so what if we keep this under the `Renderer` namespace, but split the class up into 3 then?

Reply:
> split the class up into 3 then?

Very very much agree!

Reply:
Fixed